src/host/output.cpp

```diff
@@ -520,5 +520,5 @@ void CloseConsoleProcessState()
     //      ctrl event will never actually get dispatched.
     // So, lock and unlock here, to make sure the ctrl event gets handled.
     LockConsole();
-    auto Unlock = wil::scope_exit([&] { UnlockConsole(); });
+    UnlockConsole();

```

Comment:
@zadjii-msft I've drastically simplified the PR now. In fact if we want to, we can just keep the two added `LockConsole()` lines. 🙂

Technically we can remove these two lines now, because the only place where `CloseConsoleProcessState` is called apart from these two places is in window.cpp, where it's done so under lock. As such these two lines serve no purpose anymore.